### PR TITLE
.NET 6.0

### PR DESCRIPTION
**Pull request by tibel**
use .NET 6.0 SDK and update test and benchmark dependencies

### CODE REVIEW

File: src/UnicodeNormDApp/Program.cs [hunk 1]
```diff
@@ -11,10 +11,10 @@ class Program
...
         static void Main(string[] args)
         {
-
-            var httpClient = new WebClient();
-
-            var data = httpClient.DownloadString("http://www.unicode.org/Public/UCD/latest/ucd/NormalizationTest.txt");
+            using var httpClient = new HttpClient();
+            using var response = httpClient.GetAsync("http://www.unicode.org/Public/UCD/latest/ucd/NormalizationTest.txt").Result;
+            using var content = response.Content;
+            var data = content.ReadAsStringAsync().Result;
```
Comment (MihaZupan) [inline src/UnicodeNormDApp/Program.cs:17]:
```suggestion
            var data = await new HttpClient().GetStringAsync("http://www.unicode.org/Public/UCD/latest/ucd/NormalizationTest.txt");
```
We can just make `Main` `async` here.

File: src/Markdig.Tests/Markdig.Tests.csproj [hunk 2]
```diff
@@ -1,20 +1,20 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```
Comment (MihaZupan) [inline src/Markdig.Tests/Markdig.Tests.csproj:4]: I was under the impression we had multi-targeted tests as well.
If we haven't done so before I guess keeping just 6.0 here is fine.

**Review by MihaZupan — APPROVED** [review]

**Comment by tibel** [comment]
Added another commit to fix build warnings (mostly xml doc).

File: src/global.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "5.0.100",
+    "version": "6.0.x",
```
Comment (xoofx) [inline src/global.json:3]: In the past, this was not working and required an explicit version (not like in global.json). Are you sure that it doesn't fetch the latest?

Comment (tibel) [reply src/global.json:3]: I'm not aware of this issue. Do you have a link to check if this is fixed already?
Alternatively I can change this to an explicit version, if you want.

Comment (xoofx) [inline src/global.json:3]:
```suggestion
    "version": "6.0.100",
```

Comment (xoofx) [reply src/global.json]: I don't think it is fixed as the [documentation](https://docs.microsoft.com/en-us/dotnet/core/tools/global-json?tabs=netcore3x#version) still states that you need an actual version and wildcards (e.g even `x`) are not supported

So just to be safe, easier to put an actual version. I pushed the suggestion.

**Comment by xoofx** [comment]
Thanks!

**Pull request merged**